kogito-scripts-quarkus/README.md

```diff
@@ -69,3 +80,11 @@ Complete curl command can be found below:
 ```
 curl -X POST -H 'Content-Type:application/json' -H 'Accept:application/json' -d '{"name" : "john"}' http://localhost:8080/scripts
 ```
+
+After executing the above you should see a log similar to
+
+<p align="center"><img src="docs/images/quarkusLog.png"></p>
+
+And a message after the curl command such as
```

Comment:
move the response to curl after the corresponding command, and use text instead of an image.

Reply:
Hi @evacchi made the corresponding changes for quarkus and springboot